apps/api/src/domain/models/bibliographic-reference/journal-article-bibliographic-reference/commands/create-journal-article-bibliographic-reference.command.integration.spec.ts

```diff
@@ -0,0 +1,197 @@
+import getValidBibliographicReferenceInstanceForTest from '../../../../__tests__/utilities/getValidBibliographicReferenceInstanceForTest';
+import AggregateIdAlreadyInUseError from '../../../shared/common-command-errors/AggregateIdAlreadyInUseError';
+import CommandExecutionError from '../../../shared/common-command-errors/CommandExecutionError';
+import InvalidExternalStateError from '../../../shared/common-command-errors/InvalidExternalStateError';
+import UuidNotGeneratedInternallyError from '../../../shared/common-command-errors/UuidNotGeneratedInternallyError';
+import { assertCommandFailsDueToTypeError } from '../../../__tests__/command-helpers/assert-command-payload-type-error';
+import { assertCreateCommandError } from '../../../__tests__/command-helpers/assert-create-command-error';
+import { assertCreateCommandSuccess } from '../../../__tests__/command-helpers/assert-create-command-success';
+import { DummyCommandFSAFactory } from '../../../__tests__/command-helpers/dummy-command-fsa-factory';
+import { generateCommandFuzzTestCases } from '../../../__tests__/command-helpers/generate-command-fuzz-test-cases';
+import { CommandAssertionDependencies } from '../../../__tests__/command-helpers/types/CommandAssertionDependencies';
+import { dummySystemUserId } from '../../../__tests__/utilities/dummySystemUserId';
+import { dummyUuid } from '../../../__tests__/utilities/dummyUuid';
+import { BibliographicReferenceType } from '../../types/BibliographicReferenceType';
+import { CreateJournalArticleBibliographicReference } from './create-journal-article-bibliographic-reference.command';
+import { CreateJournalArticleBibliographicReferenceCommandHandler } from './create-journal-article-bibliographic-reference.command-handler';
+
+const commandType = 'CREATE_JOURNAL_ARTICLE_BIBLIOGRAPHIC_REFERENCE';
+
+const initialState = new DeluxeInMemoryStore().fetchFullSnapshotInLegacyFormat();
+
+const dummyNewUuid = '9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dc002';
+
+const existingInstance = getValidBibliographicReferenceInstanceForTest(
+    BibliographicReferenceType.journalArticle
+).clone({
+    id: dummyUuid,
+});
+
+const testDatabaseName = generateRandomTestDatabaseName();
+
+const buildValidCommandFSA = (
+    id: AggregateId
+): FluxStandardAction<DTO<CreateJournalArticleBibliographicReference>> => ({
+    type: commandType,
+    payload: {
+        id,
+        rawData: {
+            foo: 'hello world',
+        },
+        title: 'TLDR',
+        creators: [
+            {
+                type: BibliographicSubjectCreatorType.author,
+                name: 'Dr. Johnny McPhullovit',
+            },
+        ],
+        abstract: 'Blah, blah, blah- this is the abstract.',
+        issueDate: 'Spring 2003',
+        publicationTitle: 'Metaphysical Review G',
+        url: 'https://www.myboguspublication.com/123234.pdf',
+        // TODO provide a more realistic example
+        pages: 'What is this property for? FIX ME FIX ME FIX ME',
```

Comment:
Hahah, I'll send a reminder...

Reply:
I removed this and plan to populate it in the `rawData` as discussed in a Zoom call.